src/lib/core/Global.h

```diff
@@ -85,6 +85,16 @@ class Global
     /// NOT thread-safe, external synchronization is required.
     T & get() { return _get(); }
     T * operator->() { return &_get(); }
+    template <class U = std::remove_extent_t<T>>
+    U & operator[](std::size_t i)
```

Comment:
So this is only needed for the `&sLoggingBuffer[0]` thing, right?  You could just as easily replace that with `sLoggingBuffer.get()` or `&sLoggingBuffer.get()[0]` if desired and not add the extra complexity here.

Reply:
Removed